Cargo.toml

```diff
@@ -35,15 +35,23 @@ signal-hook = "0.1.7"
 signatory = { version = "0.11", features = ["ed25519"] }
 signatory-dalek = "0.11"
 signatory-yubihsm = { version = "0.11", optional = true }
+signatory-ledger-cosval = { version = "0.11", optional = true }
 subtle-encoding = "0.3"
 tendermint = { version = "0.2", path = "tendermint-rs" }
 
+[patch.crates-io]
+signatory = { git = "https://github.com/cryptiumlabs/signatory" }
+signatory-dalek = { git = "https://github.com/cryptiumlabs/signatory" }
+signatory-yubihsm = { git = "https://github.com/cryptiumlabs/signatory" }
+signatory-ledger-cosval = { git = "https://github.com/cryptiumlabs/signatory" }
```

Comment:
please do not introduce external versions of these crates.

Reply:
This is waiting on the release of the correct upstream crates, specifically the new release of `signatory-ledger-cosval`.

Reply:
It's out: https://crates.io/crates/signatory-ledger-tm